tests/kafkatest/tests/streams/streams_broker_compatibility_test.py

```diff
@@ -20,7 +20,8 @@
 from kafkatest.services.streams import StreamsBrokerCompatibilityService
 from kafkatest.services.verifiable_consumer import VerifiableConsumer
 from kafkatest.services.zookeeper import ZookeeperService
-from kafkatest.version import LATEST_0_11_0, LATEST_0_10_2, LATEST_0_10_1, LATEST_0_10_0, LATEST_1_0, LATEST_1_1, LATEST_2_0, KafkaVersion
+from kafkatest.version import LATEST_0_11_0, LATEST_0_10_2, LATEST_0_10_1, LATEST_0_10_0, LATEST_1_0, LATEST_1_1, \
+    LATEST_2_0, LATEST_2_1, LATEST_2_2, LATEST_2_3, KafkaVersion
```

Comment:
Seems we forgot to update this for a couple of releases... If we all agree, I can open corresponding PRs for older branches to close this test gap there, too.

Reply:
Sounds like a safer approach.